code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -202,7 +202,7 @@
 	desc = "A box containing 5 highly experimental supermatter grenades."
 	icon_state = "radbox"
 	startswith = list(/obj/item/weapon/grenade/supermatter = 5)
-    
+

```

Comment:
No idea what this here's about, I checked the .DM, nothing moved above the new Atocboxes.